app/assets/stylesheets/components/shared/c-card.scss

```diff
@@ -1,14 +1,18 @@
 
 .c-card {
+  display: flex;
   position: relative;
-  margin: 0 0 20px;
+  flex-direction: column;
+  height: $card-height;
```

Comment:
Couldn't we use `align-items: stretch` then? So the card of a same line have the same height? Or the designers want _all_ the cards to have the same height?